dotnet/src/webdriver/Remote/HttpCommandExecutor.cs

```diff
@@ -286,13 +286,25 @@ private async Task<HttpResponseInfo> MakeHttpRequest(HttpRequestInfo requestInfo
                 if (_logger.IsEnabled(LogEventLevel.Trace))
                 {
                     _logger.Trace($">> {requestMessage}");
+
+                    if (requestMessage.Content != null)
+                    {
+                        string content = await requestMessage.Content.ReadAsStringAsync();
+                        _logger.Trace($">> Body: {content}");
+                    }
                 }
 
                 using (HttpResponseMessage responseMessage = await this.client.SendAsync(requestMessage).ConfigureAwait(false))
                 {
                     if (_logger.IsEnabled(LogEventLevel.Trace))
                     {
                         _logger.Trace($"<< {responseMessage}");
+
+                        if ((int)responseMessage.StatusCode < 200 || (int)responseMessage.StatusCode > 299)
```

Comment:
Hi @diemol, comment fixed. The updated value is now `399`

Reply:
The initial variant was correct, redirections are handled by lower `HttpClientHandler`, and are turned on by default (if I am not mistaken). Thus this code should never be executed in case of redirections.

Reply:
Wrote this comment and discovered another good option: if we want to log http requests/responses, we can do it in lower level... in `HttpClientHandler`. This way requires more code to be written (easy code), but it is more efficient. @ChrstnAnsl how do you feel comfortable in this topic?

Reply:
Just a clarification should Invoke this logger inside of the `CreateHttpClient` or I can create a custom Logging for the `HttpClientHandler`

Reply:
Hi @nvborisenko I agree that logging HTTP requests/responses at the HttpClientHandler level does offer better efficiency. It's a good approach if we're looking to minimize overhead and streamline the logging process. If the extra code required for logging at the lower level isn't a concern and we prioritize performance, then implementing it in HttpClientHandler is definitely a solid option. Thanks for bringing it up!

Reply:
Reviewing approach with `HttpClientHandler`, thanks for reimplementing.